### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by stvowi** [webtoolsnz/yii2-importer#2]
Is there anywhere a documentation and/or sample code to this module?

**Comment by marius-nabal** [comment]
Yes, in [UCONX (samurai repo)](https://github.com/webtoolsnz/samurai) it's used and extended a few times.  If any significant changes are to be made to it please co-ordinate with the UCONX team (especially @jmsjms) so that we can either lock in our version or fork it.  There are also probably other repos relying on this - though I don't know which ones.  I don't think any detailed docs exists for this anywhere... the included README is very thin!
